tests/sources/test_sharepoint_server.py

```diff
@@ -220,15 +220,17 @@ async def test_prepare_list_items_doc():
             "url": f"{HOST_URL}/site%5E",
             "file_name": "filename",
             "server_relative_url": "/site^",
+            "Editor": {"Title": "system"},
+            "Author": {"Title": "system"},
         }
         expected_response = {
             "type": "list_item",
             "_id": 1,
             "file_name": "filename",
             "size": 0,
             "title": "dummy",
-            "author_id": 123,
-            "editor_id": 123,
```

Comment:
This is a backwards incompatible change, if this would result in the current `author_id` and `editor_id` fields being removed. It's fine if the user who filed this issue doesn't want these fields, but that doesn't mean that no users want them or use them, so we cannot just remove them.